api/modules/feedback/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+from api.models import Feedback
+from api.modules.users.serializers import UserSerializer
+
+""" This serializer is here for sending the feedback model data in json format """
+
+
+class FeedbackIDSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Feedback
+        fields = ('id', 'text', 'type', 'created')
```

Comment:
Include `user` object in condensed response and don't have `text`

Reply:
but for the second API(the one with the id, it was mentioned to have text and no user)